docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -593,6 +593,15 @@ ALTER TABLE test_partitioned_table EXECUTE optimize
 WHERE partition_key = 1
 ```
 
+``optimize`` can also be used with ``WHERE`` clause with partitioned base 
```

Comment:
```suggestion
You can use a more complex ``WHERE`` to narrow down the scope of the ``optimize`` procedure. The following example casts the data to a date and uses a comparison to only optimize partitions with data from the year 2022 or newer:
```

Reply:
> casts the data to a date

@mosabua Is it a typo of "casts the timestamp to a date"?